services/context/context.go

```diff
@@ -184,7 +167,7 @@ func Contexter() func(next http.Handler) http.Handler {
 				}
 			})
 
-			// If request sends files, parse them here otherwise the Query() can't be parsed and the CsrfToken will be invalid.
+			// If request sends files, parse them here otherwise the Query() can't be parsed
```

Comment:
It should be a "FIXME" now, it doesn't need to `ParseMultipartForm` ahead anymore.

But removing it would cause more regressions, so leaving a comment is enough.